src/pages/Delivery/Delivery.layout.jsx

```diff
@@ -1,23 +1,15 @@
 /* eslint-disable no-unused-vars */
 // @ts-nocheck
 import PropTypes from 'prop-types'
-import { Text } from '../../components'
-import {
-  MapLayoutContainer,
-  HeaderMap,
-  HeaderText,
-  ChatBtnContainer,
-  ChatBtn,
-  HeaderTitle,
-  HeaderSubTitle,
-  FooterMap,
-  FooterMapAddress,
-  MapContainer,
-} from './Delivery.styled'
+import { Fragment } from 'react'
 import assistant from '../../assets/assistant.png'
 import chat from '../../assets/chat.png'
 import gps from '../../assets/gps-icon.png'
-import { Fragment } from 'react'
+import { Text } from '../../components'
+import {
+  ChatBtn, ChatBtnContainer, FooterMap,
+  FooterMapAddress, HeaderMap, HeaderSubTitle, HeaderText, HeaderTitle, MapContainer, MapLayoutContainer
+} from './Delivery.styled'
```

Comment:
I prefer one line for each one

```suggestion
import {
  ChatBtn, 
  ChatBtnContainer, 
  FooterMap,
  FooterMapAddress, 
  HeaderMap,
  HeaderSubTitle, 
  HeaderText, 
  HeaderTitle, 
  MapContainer, 
  MapLayoutContainer
} from './Delivery.styled'
```

Please check if is possible to run Prettier